pages/dkp/konvoy/2.2/Install/index.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: layout.pug
+navigationTitle: Install
+title: Install
+menuWeight: 80
```

Comment:
This page is kind of the bottom of the list and not very clear that should be the entrypoint for users.

<img width="360" alt="image" src="https://user-images.githubusercontent.com/4969231/165834760-7c1382e4-50b8-4148-8d41-c268bb789054.png">

Reply:
Thanks for that... forgot to change the menu weight!